src/proto/FunctionRpc.proto

```diff
@@ -262,6 +267,51 @@ message RpcFunctionMetadata {
   bool is_proxy = 7;
 }
 
+// Host tells worker it is ready to receive metadata
```

Comment:
Regarding - // Host tells worker it is ready to receive metadata - can this be a capability instead?

Reply:
Is the comment still valid? if not delete